src/SmartComponents/Overview/Dashboard.js

```diff
@@ -1,13 +1,12 @@
 /* eslint camelcase: 0 */
-import '../../App.scss';
 import './Dashboard.scss';
 
 import * as AppActions from '../../AppActions';
 
 import { ANSIBLE_MARK_ICON, GLOBAL_ECONSYSTEM_ICON, SERVER_STACK_ICON } from '../../AppSvgs';
 import { Level, LevelItem } from '@patternfly/react-core/dist/js/layouts/Level/index';
 import { PageHeader, PageHeaderTitle } from '@redhat-cloud-services/frontend-components/components/PageHeader';
-import React, { Component } from 'react';
+import React, { useEffect, useState } from 'react';
 import { Stack, StackItem } from '@patternfly/react-core/dist/js/layouts/Stack/index';
 import { global_Color_100, global_primary_color_100 } from '@patternfly/react-tokens';
```

Comment:
You should use absolute import paths here to avoid bundling the whole lib. It is available in one of the latest tokens builds.